initialized the angular app

kokobear3327 opened the pull request.

ahaly92 commented on .editorconfig:
```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```
We should rarely check in comments. The only case i can think of is TODOs and if you are stuck doing something hacky and want teammates to know why it's hacky cause it's not instantly apparent from reading code.

ahaly92 commented on .gitignore:
```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```
same for all comments across all files

ahaly92 commented on .gitignore:
```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
```
I think we can simplify this git ignore. 

/node_modules and anything that you run into locally

ahaly92 commented on README.md:
```diff
@@ -1,2 +1 @@
-# ecommerce-angular
-Here is my angular application 
+Here is my readme for my angular app
```
Readme should include a description of the app and how to run it.

ahaly92 commented on browserslist:
```diff
@@ -0,0 +1,12 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```
remove comments

ahaly92 commented on e2e/protractor.conf.js:
```diff
@@ -0,0 +1,32 @@
+// @ts-check
```
remove comments

ahaly92 commented on src/environments/environment.ts:
```diff
@@ -0,0 +1,16 @@
+// This file can be replaced during build by using the `fileReplacements` array.
```
remove comments

ahaly92 commented on tslint.json:
```diff
@@ -0,0 +1,91 @@
+  "rulesDirectory": [
+    "codelyzer"
+  ]
+}
```
files should end with a new line to avoid this weird github mark. You can probably automate this in your IDE.

ahaly92 commented on e2e/protractor.conf.js:
```diff
@@ -0,0 +1,28 @@
+    });
+    jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
+  }
+};
```
new line

ahaly92 commented on browserslist:
```diff
@@ -0,0 +1,5 @@
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```
new line

ahaly92 commented on angular.json:
```diff
@@ -0,0 +1,129 @@
+      }
+    }},
+  "defaultProject": "ecommerce-angular"
+}
```
new line

ahaly92 commented on e2e/src/app.e2e-spec.ts:
```diff
@@ -0,0 +1,22 @@
+import { AppPage } from './app.po';
```
at the moment these are just generated tests. Blow them away if they don't serve a purpose

ahaly92 commented on src/app/app.component.html:
```diff
@@ -0,0 +1,519 @@
+<style>
```
clear this file completely. This is all generated content. Just include <div>ecommerce site</div> for now

ahaly92 commented on src/app/app.component.spec.ts:
```diff
@@ -0,0 +1,35 @@
+import { TestBed, async } from '@angular/core/testing';
```
delete this test file

ahaly92 commented on src/app/app.component.ts:
```diff
@@ -0,0 +1,10 @@
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
```
delete this line and the scss file since it's completely empty

ahaly92 commented on src/environments/environment.ts:
```diff
@@ -0,0 +1,4 @@
+
+export const environment = {
+  production: false
+};
```
new line

Commit:
fix: initialized angular app and deleted comments

fix: corrected merge conflicts

fix: fixed merge conflicts

fix: removed scss file that wasn't used

ahaly92 commented:
delete src/app/app.component.scss since it's not being used. After that this should be good to go. Squash and rebase and then hit the Merge pull request button ^
